Sources/Testing/ABI/Encoded/ABI.EncodedIssue.swift

```diff
@@ -30,8 +30,17 @@ extension ABI {
     var _severity: Severity
 
     /// Whether or not this issue is known to occur.
+    ///
+    /// This should perhaps be deprecated in favor of `_knownIssueContext`.
```

Comment:
No it should not. :)

Reply:
OK, I can remove this comment. I was thinking we wouldn't want redundant data to stick around forever, but you know the design here better than I do.

Reply:
A{P,B}I is forever.

Reply:
I removed this property.